packages/app/src/cli/models/app/identifiers.ts

```diff
@@ -22,9 +22,14 @@ export interface Identifiers {
    * The extensions' numeric identifiers (expressed as a string).
    */
   extensionIds: IdentifiersExtensions
+
+  /**
+   * The extensions' unique identifiers which uuid is not managed.
+   */
+  extensionsNonUuidManaged: IdentifiersExtensions
```

Comment:
What if we call this `configurationIds`?

Reply:
I wanted to introduce the concept `uuidManaged` (right now only extensions that includes `app_config` feature) as extensions which `uuids` are not going to be neither matched nor stored in the `.env` file. Maybe in the future we should extension the method `isUuidManaged` to include a different condition, so all the extensions that matches that condition will have the same behaviour. That's why this is the list of `non uuid managed`

Reply:
I just prefer a clearer name, even if it's not so generic. But it's only a name, let's move on!